c_emulator/riscv_sim.cpp

```diff
@@ -385,7 +404,12 @@ uint64_t load_sail(char *f, bool main_file)
   bool is32bit;
   uint64_t entry;
   uint64_t begin_sig, end_sig;
-  load_elf(f, &is32bit, &entry);
+  if (load_offset) {
+    // dynamic load
```

Comment:
Uh ET_EXEC should ignore it

Reply:
do you mean we load an `fw_jump.elf` then with a kernel?

Reply:
fw_jump vs fw_dynamic is an orthogonal concern (referring to whether OpenSBI hard-codes the location at which the payload is loaded) to whether the kernel being loaded requests a fixed address (that may or may not match OpenSBI's expectation).

Reply:
currently the sail gives no api on detect whether it is DYN or EXEC.